instrumentation/kamon-system-metrics/src/main/scala/kamon/instrumentation/system/host/HostMetrics.scala

```diff
@@ -4,127 +4,145 @@ import kamon.Kamon
 import kamon.instrumentation.system.host.HostMetrics.StorageDeviceInstruments.DeviceInstruments
 import kamon.instrumentation.system.host.HostMetrics.StorageMountInstruments.MountInstruments
 import kamon.instrumentation.system.host.HostMetrics.NetworkActivityInstruments.InterfaceInstruments
-import kamon.metric.{Counter, Gauge, InstrumentGroup, MeasurementUnit}
+import kamon.metric.{Counter, Gauge, Histogram, InstrumentGroup, MeasurementUnit}
 import kamon.tag.TagSet
 
 import scala.collection.mutable
 
 object HostMetrics {
 
-  val CpuUsage = Kamon.histogram (
+  val CpuUsage = Kamon.histogram(
     name = "host.cpu.usage",
     description = "Samples the CPU usage percentage",
     unit = MeasurementUnit.percentage
   )
 
-  val MemoryUsed = Kamon.gauge (
+  val MemoryUsed = Kamon.gauge(
     name = "host.memory.used",
     description = "Tracks the amount of used memory",
     unit = MeasurementUnit.information.bytes
   )
 
-  val MemoryFree = Kamon.gauge (
+  val MemoryUsage = Kamon.histogram(
+    name = "host.memory.usage",
+    description = "Tracks the percetange of memory used",
+    unit = MeasurementUnit.percentage
+  )
+
+  val MemoryFree = Kamon.gauge(
     name = "host.memory.free",
     description = "Tracks the amount of free memory",
     unit = MeasurementUnit.information.bytes
   )
 
-  val MemoryTotal = Kamon.gauge (
+  val MemoryTotal = Kamon.gauge(
     name = "host.memory.total",
     description = "Tracks the total memory available",
     unit = MeasurementUnit.information.bytes
   )
 
-  val SwapUsed = Kamon.gauge (
+  val SwapUsed = Kamon.gauge(
     name = "host.swap.used",
     description = "Tracks the used Swap space",
     unit = MeasurementUnit.information.bytes
   )
 
-  val SwapFree = Kamon.gauge (
+  val SwapUsage = Kamon.histogram(
+    name = "host.swap.usage",
+    description = "Tracks the percentage of swap used",
+    unit = MeasurementUnit.percentage
+  )
+
+  val SwapFree = Kamon.gauge(
     name = "host.swap.free",
     description = "Tracks the free Swap space",
     unit = MeasurementUnit.information.bytes
   )
 
-  val SwapTotal = Kamon.gauge (
+  val SwapTotal = Kamon.gauge(
     name = "host.swap.total",
     description = "Tracks the total Swap space",
     unit = MeasurementUnit.information.bytes
   )
 
-  val LoadAverage = Kamon.gauge (
+  val LoadAverage = Kamon.gauge(
     name = "host.load.average",
     description = "Tracks the system load average"
   )
 
-  val FileSystemMountSpaceUsed = Kamon.gauge (
+  val FileSystemMountSpaceUsed = Kamon.gauge(
     name = "host.storage.mount.space.used",
     description = "Tracks the used space on a file system mount/volume",
     unit = MeasurementUnit.information.bytes
   )
 
-  val FileSystemMountSpaceFree = Kamon.gauge (
+  val FileSystemMountSpaceUsage = Kamon.histogram(
```

Comment:
This metric should be a gauge, same as the `host.storage.mount.space.used|free|total` metrics.